versioned_docs/version-v1.2/vm/backup-restore.md

```diff
@@ -171,3 +171,13 @@ You can only choose to retain the previous volumes.
 1. Click `Replace Existing`.
 1. You can view the restore process from the `Virtual Machines` page.
 ![restore-vm-snapshot-existing.png](/img/v1.2/vm/restore-vm-snapshot-existing.png)
+
+## Known issues
+
+### VM Backup metadata file name conflicts
+
+When users create a VM Backup, harvester creates a metadata file in the [backup target](../advanced/settings.md#backup-target). The metadata file keeps VM Backup YAML data. The metadata file path is `<storage-path>/harvester/vmbackups/<vmbackup-namespace>-<vmbackup-name>.cfg`.
+
+If users create a VM Backup `c` in `a-b` namespace and another `b-c` in `a` namespace. The metadata file name conflicts, because both are `a-b-c.cfg`. The issue will be fixed in v1.3.0. The metadata file path will be `<storage-path>/harvester/vmbackups/<vmbackup-namespace>/<vmbackup-name>.cfg`.
+
+In a havester version before v1.3.0, users should avoid creating VM Backups which may make the metadata file name conflict.
```

Comment:
```suggestion
### VM Backup Metadata File Naming Conflicts

Whenever you create a VM backup, Harvester generates a metadata file in the [backup target](../advanced/settings.md#backup-target). The metadata file, which is found in `<storage-path>/harvester/vmbackups/<vmbackup-namespace>-<vmbackup-name>.cfg`, contains VM backup data in YAML format.

The naming convention for these metadata files can introduce conflicts. Specifically, files generated for VM backups that were created in different namespaces can have the exact same file name.

Example:
| VM backup name | Namespace | Metadata file name |
| --- | --- | --- |
| `c` | `a-b` | `a-b-c.cfg` |
| `b-c` | `a` | `a-b-c.cfg` |

Harvester v1.3.0 fixes this issue by changing the metadata file path to `<storage-path>/harvester/vmbackups/<vmbackup-namespace>/<vmbackup-name>.cfg`. If you are using an earlier version, however, ensure that VM backup names do not cause the described file naming conflicts.
```

Reply:
Hi @jillian-maroket, thanks for the review. I updated the document, but changing one line. The data is stored in JSON format.